src/components/DayOverview/DayOverviewPdf.tsx

```diff
@@ -0,0 +1,202 @@
+"use client";
+
+import React from "react";
+import { Svg, Document, Page, Text, View, StyleSheet, Path } from "@react-pdf/renderer";
+import { faFlag, faSquare, IconDefinition } from "@fortawesome/free-solid-svg-icons";
+
+import { ParcelOfSpecificDateLocation } from "@/components/DayOverview/DayOverview";
+interface DayOverviewerPdfProps {
+    date: Date;
+    location: string;
+    data: ParcelOfSpecificDateLocation[];
+}
+
+interface DayOverviewerCardProps extends DayOverviewerPdfProps {}
+
+interface GeneralCellStyle {
+    borderStyle: "solid" | "dashed" | "dotted" | undefined;
+    borderWidth: number;
+    borderLeftWidth: number;
+    borderTopWidth: number;
+    padding: string;
+}
+
+interface CustomSVGProps {
+    icon: IconDefinition;
+    color: string;
+    fill: boolean;
+}
+
+const generalCellStyle: GeneralCellStyle = {
```

Comment:
So how it is done in the docs/ how we've done it elsewhere is very different. Instead of using consts, we combine classes like `[styles.class1, styles.class2, { any inline styles}]`. Have a look at mine or Sam's code perhaps for how we've done it.

Reply:
Done